lnrpc/routerrpc/router_server.go

```diff
@@ -484,12 +484,16 @@ func (s *Server) QueryMissionControl(ctx context.Context,
 // toRPCPairData marshalls mission control pair data to the rpc struct.
 func toRPCPairData(data *routing.TimedPairResult) *PairData {
 	rpcData := PairData{
-		AmtSat:                int64(data.Amt.ToSatoshis()),
-		LastAttemptSuccessful: data.Success,
+		FailAmtSat:    int64(data.FailAmt.ToSatoshis()),
+		SuccessAmtSat: int64(data.SuccessAmt.ToSatoshis()),
 	}
 
-	if !data.Timestamp.IsZero() {
-		rpcData.Timestamp = data.Timestamp.Unix()
+	if !data.FailTime.IsZero() {
+		rpcData.FailTime = data.FailTime.Unix()
```

Comment:
Won't calling `Unix()` on zero time just be zero anyway?

Reply:
No, zero time is somewhere far before 1970.